packages/main/test/sap/ui/webcomponents/main/qunit/runner.html

```diff
@@ -56,7 +56,9 @@
 			var params = getUrlQueryParams();
 
 			insertScript(params.testName + ".qunit.js");
-		});
+		};
+
+		loadTests();
```

Comment:
Shouldn't we still wait for document.ready to run the tests? maybe that's why they're failing?